content/docs/secure/encryption/deployment.md

```diff
@@ -51,15 +71,18 @@ encryption:
+| image | Encryption driver image name.                                                                                                                                                                                                                                                                                                                                                                 | No       | "dellemc/csm-encryption:v0.1.0" |
+| imagePullPolicy | If specified, overrides the chart global imagePullPolicy.                                                                                                                                                                                                                                                                                                                                     | No       | CSI driver global imagePullPolicy |
+| logLevel | Log level of the encryption driver.<br/>Allowed values: "error", "warning", "info", "debug, `"trace".                                                                                                                                                                                                                                                                                         | No       | "error" |
+| apiPort | TCP Port number used by the REST API Server.                                                                                                                                                                                                                                                                                                                | No       | 3838 |
+| livenessPort | HTTP liveness probe port number. Leave empty to disable the liveness probe.                                                                                                                                                                                                                                                                                                                   | No       | |
+| ocp | Enable when running an OCP Platform with CoreOS worker nodes.                                                                                                                                                                                                                                 | No       | false |
+|ocpCoreID| User ID and group ID of user core on CoreOS worker nodes. Ignored when ocp is set to false.  | No       | "1000:1000" |
+| extraArgs | Extra command line parameters to pass to the encryption driver.<br/>Allowed values:<br/>"\-\-sharedStorage" - may be required by some applications to work properly.<br/>When set, performance is reduced and hard links cannot be created.<br/>See the [gocryptfs documentation](https://github.com/rfjakob/gocryptfs/blob/v2.2.1/Documentation/MANPAGE.md#-sharedstorage) for more details. | No       | [] |

```

Comment:
Can you make it compact as it was before?
E.g.
```
| --------- | ----------- | -------- | ------- |  
```
instead of
```
| --------- |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------| ------- |
```
Or
```
| enabled | Enable/disable volume encryption feature. | No | false |
```
instead of
```
| enabled | Enable/disable volume encryption feature.                                                                                                                                                                                                                                                                                                                                                     | No       | false |
```
And do the same for other modified tables.